core/src/main/java/com/novoda/noplayer/Player.java

```diff
@@ -31,7 +31,7 @@
 
     void attach(PlayerView playerView);
 
-    void selectAudioTrack(int audioTrackIndex);
+    void selectAudioTrack(PlayerAudioTrack audioTrack);
```

Comment:
Introduced type safety rather than allowing clients to pass back any integer that could potentially crash the `Player`.